source/includes/auto-encryption-options.rst

```diff
@@ -0,0 +1,128 @@
+The following table describes the structure of an
+``AutoEncryptionOptions`` object:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 30 70
+
+   * - Name
+     - Description
+
+   * - **keyVaultClient**
+     - **Type:** ``MongoClient``
+
+       **Description**
+       A ``MongoClient`` configured to connect to
+       the MongoDB instance hosting your {+key-vault-long+}. For more
+
+       If you omit the ``keyVaultClient`` option, the driver uses the
+       MongoDB instance specified to your ``MongoClient`` containing the
+       ``AutoEncryptionOpts`` configuration as the host of your key vault
+       collection.
+
+       To learn more about {+key-vault-long+}s, see :ref:`qe-reference-key-vault`.
+
+       **Default:** ``undefined``
+       **Accepted Values:** A configured ``MongoClient``
```

Comment:
Suggestion: separate these two lines to make it easier to scan (using the "|" rST formatter).

Reply:
E.g. https://github.com/mongodb/docs-kafka-connector/blob/master/source/source-connector/configuration-properties/change-stream.txt#L39-L43